openwisp_users/admin.py

```diff
@@ -271,6 +275,43 @@ def has_change_permission(self, request, obj=None):
             return False
         return super().has_change_permission(request, obj)
 
+    def get_actions(self, request):
+        actions = super().get_actions(request)
+        if not request.POST.get('post') and 'delete_selected' in actions:
+            del actions['delete_selected']
+        return actions
+
+    def delete_selected_modified(self, request, queryset):
+        if not request.user.is_superuser:
+            org_owners = OrganizationOwner.objects.filter(
+                organization_id__in=request.user.organizations_pk
+            ).values_list('organization_user__user', flat=True)
+            qs = queryset.exclude(pk__in=org_owners)
+            if queryset.count() and not qs.count():
+                self.message_user(
+                    request, _('Can not delete organization owners'), messages.ERROR
```

Comment:
`Can not delete organization owners: {users}.`

Make users comma separated list of usernames, eg: 'user1, user2'.

Use something like `', '.join(username_list)`